samples/debye_hueckel.py

```diff
@@ -18,7 +18,7 @@
 #
 from __future__ import print_function
 import numpy
-import espressomd._system as es
+import espressomd.system as es
```

Comment:
`es.*` is not used anywhere in that script. It only is part of outdated comments.
So please just delete this line. Also the commented lines containing the outdated commands.